contract_comm/evm.go

```diff
@@ -41,22 +41,95 @@ type InternalEVMHandler struct {
 	chain vm.ChainContext
 }
 
+// MakeStaticCallToAddress performs a static (read-only) ABI call to the smart contract address given.
```

Comment:
Keeping this in the same place as the previous function will make reviewing easier